src/pages/Encounters/EncounterList.tsx

```diff
@@ -346,32 +340,32 @@ export function EncounterList({
                     }}
                   >
                     <SelectTrigger className="h-8 w-[120px]">
-                      <SelectValue placeholder="Status" />
+                      <SelectValue placeholder={t("status")} />
                     </SelectTrigger>
                     <SelectContent>
-                      <SelectItem value="all">All Status</SelectItem>
-                      <SelectItem value="planned">
+                      <SelectItem value="all">{t("all_status")}</SelectItem>
```

Comment:
should do the same thing as above (priorities), instead of individually defining the statuses.